### ISSUE TITLE
[BUG] Uninstallation notification says `Installed`

### ISSUE THREAD

**Issue by opk12** [Droid-ify/client#513]
When an app is uninstalled (removed), the notification says `Installed`.

**Comment by Iamlooker** [comment]
What version is this?

**Comment by vdbhb59** [comment]
I faced it in the last 2 F-Droid released versions.

**Comment by omentic** [comment]
I have encountered the opposite issue -- when installing some apps sometimes, I've gotten notifications saying "Uninstalled".

**Comment by vdbhb59** [comment]
UI bug?

**Comment by omentic** [comment]
Yes, very minor thing. It's probably whatever this is. Some incorrect check somewhere maybe?
